src/stores/widgets/ElementWidgetActions.ts

```diff
@@ -37,6 +35,23 @@ export enum ElementWidgetActions {
      * @deprecated Use MSC2931 instead
      */
     ViewRoom = "io.element.view_room",
+    // This can be sent as from or to widget
+    // fromWidget: updates the client about the current device mute state
+    // toWidget: the client requests a specific device mute configuration
+    //   The reply will always be the resulting configuration
+    //   It is possible to sent an empty configuration to retrieve the current values or
+    //   just one of the fields to update that particular value
+    //   An undefined field means that EC will keep the mute state as is.
+    //   -> this will allow the client to only get the current state
+    //
+    // The data of the widget action request and the response are:
+    // {
+    //   audio_enabled?: boolean,
+    //   video_enabled?: boolean
+    // }
+    // NOTE: this is currently unused. Its only here to make EW aware
+    // of this action so it does not throw errors.
```

Comment:
I wonder if this might get confusing having the same event type have two different meanings? Would it be better to separate announcing the mute status from setting it?

Reply:
I was considering this. But i think looking at an action as the full request: `toWidget:device_mute` (or `toWidget:send_event`, `fromWidget:send_event`) makes sense and is what we do with other widget actions as well.

Reply:
ah ok, well consistency ftw then